Form-Controls/README.md

```diff
@@ -31,23 +31,23 @@ Do not write a form action for this project.
 
 Let's write out our testable criteria:
 
-- [ ] I have used HTML and CSS only.
+- [ * ] I have used HTML and CSS only.
 
 ### HTML
 
-- [ ] My form is semantic html.
-- [ ] All inputs have associated labels.
-- [ ] My Lighthouse Accessibility score is 100.
-- [ ] I require a valid name. I have defined a valid name as a text string of two characters or more.
-- [ ] I require a valid email.
-- [ ] I require one colour from a defined set of 3 colours.
-- [ ] I require one size from a defined set of 6 sizes.
-- [ ] I require one date from a constrained date range.
+- [ * ] My form is semantic html.
```

Comment:
Pro tip, task lists on GitHub have special powers. If you put an X inside, it will turn into a checkbox

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/about-task-lists

- [ ] Write a list on GitHub
- [ ] Choose the checkbox list from the rich text editor
- [ ] Look at the preview pane
- [x] Check an item
- [ ] Look back at the preview pane